### DISCUSSION TITLE
Sonoff Touch Tasmota connects to Powerline?

### DISCUSSION THREAD

**Discussion by pascal260303** [arendst/Tasmota#16931] [General]
Hello
I got a Sonoff Touch with Tasmota and I have a Fritz Powerline Adapter in my local network and a Fritz Box. Now sometimes when I look at the connected devices on the Fritz Box, the Sonoff is sometime displayed as connected via Lan to the Powerline Adapter. Is there really a way this is how its connected or is that a bug?
Regards
Pascal Kuppler

**Comment by barbudor** [comment]
If I understand the powerline adapter is a secondary wifi access point connected to the fritzbox through powerline

Is you have setoption 57 on, tasmota is free to jump to what appears as the strongest access point anytime (actually every 44 minutes is I remember well)
Depending on your building, furniture, where the Sonoff is located, people moving around, that's still possible
